typeshed/2.7/galaxy/tools/deps/__init__.pyi

```diff
@@ -0,0 +1,41 @@
+# Stubs for galaxy.tools.deps (Python 3.4)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from typing import Any, Optional
+from .requirements import ToolRequirement as ToolRequirement, ToolRequirements as ToolRequirements
+from .resolvers import NullDependency as NullDependency
+from .resolvers.conda import CondaDependencyResolver as CondaDependencyResolver
+from .resolvers.galaxy_packages import GalaxyPackageDependencyResolver as GalaxyPackageDependencyResolver
+from .resolvers.tool_shed_packages import ToolShedPackageDependencyResolver as ToolShedPackageDependencyResolver
+
+log = ...  # type: Any
+CONFIG_VAL_NOT_FOUND = ...  # type: Any
+
+def build_dependency_manager(config): ...
```

Comment:
This needs a full type signature. It returns a `DependencyManager`, but I am unsure of what type `config` is.